src/core_plugins/kibana/public/discover/partials/no_results.html

```diff
@@ -28,19 +36,35 @@
     </div>
 
     <div ng-show="opts.timefield">
+      <h3>
+        Expand your time range
+      </h3>
+
       <p>
-        <h3>Expand your time range</h3>
-        <p>I see you are looking at an index with a date field. It is possible your query does not match anything in the current time range, or that there is no data at all in the currently selected time range. Click the button below to open the time picker. For future reference you can open the time picker by clicking on the <a class="kuiButton kuiButton--primary kuiButton--small" ng-click="kbnTopNav.toggle('filter')" aria-expanded="kbnTopNav.is('filter')" aria-label="time picker" data-test-subj="discoverNoResultsTimefilter"><span aria-hidden="true" class="kuiIcon fa-clock-o"></span> time picker</a> button in the top right corner of your screen.
+        I see you are looking at an index with a date field. It is possible your query does not match anything in the current time range, or that there is no data at all in the currently selected time range. Click the button below to open the time picker. For future reference you can open the time picker by clicking on the
+        <a
+          class="kuiButton kuiButton--primary kuiButton--small"
+          ng-click="kbnTopNav.toggle('filter')"
+          kbn-accessible-click
```

Comment:
I guess you already discussed this, but can't find it right now: wouldn't it make sense to add [`ngAria`](https://docs.angularjs.org/api/ngAria) as a dependency, to solve that basic issues (like automatically make ng-click stuff tabable and clickable by keyboard)?